torn/plugins/console.py

```diff
@@ -1,8 +1,25 @@
 """Module handles torn_cli actions and does the needful"""
 import os
+import requests
+import zipfile
+__version__ = "0.0.4"
+
 def handler(args):
     if args.action[0] == 'run':
         runHandler()
+    elif args.action[0] == 'new':
+        #  for new app , cloning form main repository
+        print('creating new app...')
```

Comment:
Please make a separate function, like runHandler on L#9, for this section, for example: newHandler and then call it here

Reply:
okay , will do in that way. thanks btw